Bill3.java

```diff
@@ -3,6 +3,10 @@ public class Bill3 {
 	public static void main(String[] args) {
 		// To get you started, here is the first line in the program:
 	    String name1 = args[0];
-	    // Replace this comment with the rest of your code   
+	    String name2 = args[1];
+		String name3 = args[2];
+		int bill_sum = Integer.parseInt(args[3]);
```

Comment:
In java it is common to use camelCase (in python we use underscores)
bill_sum -> billSum